styles/styles.css

```diff
@@ -0,0 +1,112 @@
+body {
+  background-color: #FFFFFF;
+}
+
+.container {
+  width: 960px;
+  margin: 0 auto;
+  background-color: gray;
+}
+
+.main-header {
+  height: 130px;
+  background-color: green;
+}
+
+.logo {
+  height: 60px;
+  width: 60px;
+  background-color: gray;
+  position: absolute;
+  top: 0;
+  left: 47%;
```

Comment:
I am not counting you off for this, because I didn't explain it until after you turned in the assignment, but if you want to center an element with `position: absolute`, you need to set `left: 50%` and then add a **negative margin of half the width**.

For example:

```css
.logo {
  height: 60px;
  width: 60px;
  background-color: gray;
  position: absolute;
  top: 0;
  left: 50%;
  margin-left: -30px;
}
```